docs/book/src/reference/kind.md

```diff
@@ -0,0 +1,52 @@
+# Kind Cluster
+
+This only cover the basics to use a kind cluster. You can find more details at
+[kind documentation](https://kind.sigs.k8s.io/).
+
+## Installation
+
+You can follow [this](https://kind.sigs.k8s.io/#installation-and-usage) to
+install `kind`.
+
+## Create a Cluster
+
+You can simply create a `kind` cluster by
+
+```bash
+kind create cluster
+```
+
+To customize your cluster, you can provide additional configuration.
+For example, the following is a sample `kind` configuration.
+
+```yaml
+{{#include ./testdata/project/hack/kind-config.yaml}}
```

Comment:
This is not being rendered properly. Not sure what is missing.

Reply:
Good catch.
This issue is caused by moving this doc from `tutorial` to `reference`.